src/repository/DeviceAttributeDatabaseRepository.js

```diff
@@ -46,8 +46,11 @@ class DeviceAttributeDatabaseRepository extends BaseRepository
     deviceIDs: Array<string>,
     ownerID?: string,
   ): Promise<Array<DeviceAttributes>> =>
+    // todo  $in operator doesn't work for neDb(no matter with regexp or plain strings)
     (await this._database.find(this._collectionName, {
-      deviceID: { $in: deviceIDs },
+      deviceID: {
+        $in: deviceIDs.map((id: string): RegExp => new RegExp(`^${id}$`, 'i')),
```

Comment:
What is this doing?

Reply:
case insensitive search, so search by id '123a' will find '123A' and '123a'
there are better ways to do it in the latest mongodb, but we don't use the latest.